stories/index.js

```diff
@@ -17,11 +17,19 @@ import KatPersona from './components/KatPersona';
 import WarningMessage from './components/WarningMessage';
 
 import './index.css';
-import choreography from '../src/factory/choreography';
+import choreography from '../src/factory';
 import { opacity, translate, rotate, scale } from '../src/presets/index';
 
 const BatmanWipeTransition = choreography([opacity, rotate, scale.all]);
 
+const SwipeInTransition = choreography([{
+  transition: 'max-width',
```

Comment:
@Setsun best yet I added an example.

CSS uses the dasherized name: `transition: max-width .3s ease-in` but when you set the `style` prop you must use the camelCase `style={{ maxWidth: '400px' }}`

The react warning is `Warning: Unsupported style property z-index. Did you mean zIndex?`